src/Sarif/Schemata/sarif-external-property-file.json

```diff
@@ -0,0 +1,132 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "SARIF External Property File Schema Version 2.0.0-csd.2.beta.2019-04-03 JSON Schema",
+    "type": "object",
+    "properties": {
+
+        "schema": {
+            "description": "The URI of the JSON schema corresponding to the version of the external property file format.",
+            "type": "string",
+            "format": "uri"
+        },
+
+        "version": {
+            "description": "The SARIF format version of this external properties object.",
+            "enum": [ "2.0.0-csd.2.beta.2019-04-03" ]
+        },
+
+        "guid": {
+            "description": "A stable, unique identifer for this external properties object, in the form of a GUID.",
+            "type": "string"
+        },
+
+        "runGuid": {
+            "description": "A stable, unique identifer for the run associated with this external properties object, in the form of a GUID.",
+            "type": "string"
+        },
+
+        "conversion": {
+            "description": "A conversion object that will be merged with an external run.",
+            "$ref": "http://json.schemastore.org/sarif#/definitions/conversion"
```

Comment:
>http://json.schemastore.org/sarif [](start = 21, length = 33)

instead use [https://schemastore.azurewebsites.net/json/sarif](https://schemastore.azurewebsites.net/json/sarif) here and elsewhere